tests/test_trace_exporter.py

```diff
@@ -174,4 +181,14 @@ def test_links(self, mock_tracer):
     @patch("lightstep.Tracer")
     def test_resources(self, mock_tracer):
         """ test resources """
-        # TODO
+        resource = trace.Resource(labels={"test": "123", "other": "456"})
+        otel_span = trace.Span(
+            name=__name__, context=self._span_context, resource=resource,
+        )
+        otel_span.set_attribute("test", "789")
+        otel_span.set_attribute("one-more", "000")
+        result_spans = self._process_spans([otel_span])
+        self.assertEqual(len(result_spans), 1)
+        self.assertEqual(result_spans[0].tags.get("test"), "789")
+        self.assertEqual(result_spans[0].tags.get("other"), "456")
+        self.assertEqual(result_spans[0].tags.get("one-more"), "000")
```

Comment:
```suggestion
        self.assertEqual(result_spans[0].tags["test"], "789")
        self.assertEqual(result_spans[0].tags["other"], "456")
        self.assertEqual(result_spans[0].tags["one-more"], "000")
```
Better to get a `KeyError` than an `AssertionError` because `"789" is not None`.